### ISSUE TITLE
[Feedback] Tutorial does not build! Plugins do not add 

### ISSUE THREAD

**Issue by mafudge** [aws-amplify/docs#3102]
**Page**: [`/start/getting-started/add-api/q/integration/flutter`](https://docs.amplify.aws/start/getting-started/add-api/q/integration/flutter#setup-aws-cloud-resources-with-amplify-cli)

**Feedback**:

followed this tutorial ... nada

Amplify plugin was not added
```bash
Error: AmplifyException$(message: Amplify plugin AmplifyAuthCognito was not added successfully., recoverySuggestion: We currently don't have a recovery suggestion for this exception., underlyingException: MissingPluginException(No implementation found for method addPlugin on channel com.amazonaws.amplify/auth_cognito))
    at Object.throw_ [as throw] (http://localhost:37149/dart_sdk.js:5334:11)
    at amplify.AmplifyClass._private.addPlugin (http://localhost:37149/packages/amplify_flutter/amplify_hub.dart.lib.js:254:23)
    at addPlugin.throw (<anonymous>)
    at http://localhost:37149/dart_sdk.js:39038:38
    at _RootZone.runBinary (http://localhost:37149/dart_sdk.js:38894:58)
    at _FutureListener.thenAwait.handleError (http://localhost:37149/dart_sdk.js:33887:48)
    at handleError (http://localhost:37149/dart_sdk.js:34451:51)
    at Function._propagateToListeners (http://localhost:37149/dart_sdk.js:34477:17)
    at _Future.new.[_completeWithValue] (http://localhost:37149/dart_sdk.js:34315:23)
    at async._AsyncCallbackEntry.new.callback (http://localhost:37149/dart_sdk.js:34338:35)
    at Object._microtaskLoop (http://localhost:37149/dart_sdk.js:39176:13)
    at _startMicrotaskLoop (http://localhost:37149/dart_sdk.js:39182:13)
    at http://localhost:37149/dart_sdk.js:34689:9
Error: MissingPluginException(No implementation found for method configure on channel com.amazonaws.amplify/amplify)
    at Object.throw_ [as throw] (http://localhost:37149/dart_sdk.js:5334:11)
    at MethodChannel._invokeMethod (http://localhost:37149/packages/flutter/src/services/system_channels.dart.lib.js:954:21)
    at _invokeMethod.next (<anonymous>)
    at http://localhost:37149/dart_sdk.js:39032:33
    at _RootZone.runUnary (http://localhost:37149/dart_sdk.js:38889:58)
    at _FutureListener.thenAwait.handleValue (http://localhost:37149/dart_sdk.js:33875:29)
    at handleValueCallback (http://localhost:37149/dart_sdk.js:34435:49)
    at Function._propagateToListeners (http://localhost:37149/dart_sdk.js:34473:17)
    at _Future.new.[_completeWithValue] (http://localhost:37149/dart_sdk.js:34315:23)
    at async._AsyncCallbackEntry.new.callback (http://localhost:37149/dart_sdk.js:34338:35)
    at Object._microtaskLoop (http://localhost:37149/dart_sdk.js:39176:13)
    at _startMicrotaskLoop (http://localhost:37149/dart_sdk.js:39182:13)
    at http://localhost:37149/dart_sdk.js:34689:9
```

**Comment by medelman17** [comment]
Hi! Can you provide a bit more context? For example, at which step in the tutorial did things start to break down?

**Comment by mafudge** [comment]
Of course. I completed the tutorial from The beginning until "next steps"  https://docs.amplify.aws/start/getting-started/nextsteps/q/integration/flutter. I'm am using linux  for my OS and vs code for my IDE

Before I began any of those additional steps in the "Next steps" section. 

I ran the code and this is the exception/stack trace I got. For some reason it seems like the plugin architecture is not registering the configuration. 

I've tried the usual stuff like flutter clean, and running the example outside of vs code. 

When I disable all plugins, just using Amplify.configure() I still get  the 2nd exception.
